docs/basics/internal.md

```diff
@@ -85,6 +85,7 @@ q)-4!"a variable named aa_bb"
 q)
 ```
 
+!!! warning Should not be used with input data over 2gb in length (0Wi). Returns domain error with this condition since 4.1 2022.04.15.
```

Comment:
@sshanks-kx, is this 2 gigabytes? If so, the std unit should be GB.

Reply:
changes now made